macro-plantuml-macro/src/main/java/org/xwiki/contrib/plantuml/PlantUMLMacroParameters.java

```diff
@@ -50,6 +62,14 @@ public String getServer()
         return this.serverURL;
     }
 
+    /**
+     * @return the (optional) PlantUML output format, png by default. Valid values: svg, png, txt
```

Comment:
Same as above. No need to dupicate. Better put a `See {@link #getFormat()}`  above and document it here properly.